assignment-6/src/Components/index.js

```diff
@@ -0,0 +1,5 @@
+import CardComponent from "./CardContainer";
```

Comment:
use index.js in case you have CardContainer's .css also or constants also but since you have only 1 file here. You can drop using index.js here for now.
the Structure would be :
components ------ CardContainer(folder) --- CardContainer.js --- CardContainer.css -- index.js ( with both two of the files)